WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -485,14 +531,16 @@ runif(7, min=10, max=100)
 
 ### sample()
 
-`sample(x, n, replace=TRUE/FALSE)` returns a sample of `n` integers taken in the object `x` (which may be a vector, a series such as `1:100`, etc). `replace=` followed by either TRUE or FALSE defines whether or not a data element can appear repeatedly in the sample.
+`sample(x, n, replace=TRUE/FALSE)` returns a sample of `n` integers taken in the object `x` (which may be a vector, a series such as `1:100`, etc). 
+`replace=` followed by either TRUE or FALSE defines whether or not a data element can appear repeatedly in the sample.
 
 ```{r sample, echo=TRUE}
 sample(1:100, 10, replace=FALSE)
 sample(20:30, 7, replace=TRUE)
 ```
 
-An interesting property of the function `sample()` is that it can be used to shuffle a vector, something which is useful for randomization of data elements. In the following example, `sample()` shuffles and returns all values in the vector `one.to.ten`:
+An interesting property of the function `sample()` is that it can be used to shuffle a vector, something which is useful for randomization of data elements. 
```

Comment:
the term vector has not been introduced yet. maybe link to later or explain, a series of numbers...